source/io/PropertyNode.cpp

```diff
@@ -15,18 +15,22 @@ SDG
 #include "ExecutionContext.h"
 #include "StringUtilities.h"
 #include "TDCodecVia.h"
+#include "VirtualInstrument.h"
 #include <stdio.h>
 
+
+#define VIREO_TYPE_PropertyNode 1
```

Comment:
Suggest renaming this; this name just doesn't 'sound' like an token/enabling macro.

Reply:
I saw this convention being used in a couple of other files. What is the recommended style?

Reply:
That form, with VIREO_TYPE_ prefix, is supposed to only be used for guarding inclusion of support of Vireo primitive data types, not other features or primitives.
However, I notice that JSLI, HttpClient, and several others have already used this form as a guard for features instead of types, so I retract my objection.  (Maybe this should be a tech debt item to clean this up.)

Reply:
Perhaps we can change this one now and do the others later. Use VIREO_MODULE_PropertyNode, instead. The idea is that you can compile modules out as required by a specific target. The idea was to implement HTTP and JSLI as Vireo Modules.